### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andresroberto**
Update title and author email in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
-# proyecto-del-murcielago
+# Proyecto del murciélago
 
 Este repositorio es para practicar el flujo de Github
```
Comment (Ginagonzalezgut) [inline README.md:3]: Could you add a dot at the end of the paragraph?

File: README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+
 Estoy aprendiendo:
```
Comment (Ginagonzalezgut) [inline README.md:6]: Could you add javaScript?

File: README.md (hunk 1 above)
Comment (andresroberto) [reply README.md:3]: Sure

File: README.md (hunk 2 above)
Comment (andresroberto) [reply README.md:6]: Done

**Review by Ginagonzalezgut — APPROVED** [review]
Good job!

**Pull request merged**